postprocessing_variant_calls/maf/tag/tag_process.py

```diff
@@ -268,5 +268,171 @@ def traceback(
+#         callback=check_separator,
+#     ),
+# ):
+#     # prep maf
+#     mafa = MAFFile(maf, separator)
+#     typer.secho(
+#         f"Tagging MAF with column for presence of exonic variant", fg=typer.colors.BRIGHT_GREEN
+#     )
+#     mafa = mafa.tag_exonic_variant()
+#     typer.secho(f"Writing Delimited file: {output_maf}", fg=typer.colors.BRIGHT_GREEN)
+#     mafa.to_csv(f"{output_maf}".format(outputFile=output_maf), index=False, sep="\t")
+#     return 0
+
+
+# @app.command(
+#     "MET_variant", help="Tag a variant in a MAF file if it is classified as a MET variant."
+# )
+# def MET_variant(
+#     maf: Path = typer.Option(
+#         ...,
+#         "--maf",
+#         "-m",
+#         exists=True,
+#         file_okay=True,
+#         dir_okay=False,
+#         writable=False,
+#         readable=True,
+#         resolve_path=True,
+#         help="MAF file to tag",
+#     ),
+#     output_maf: Path = typer.Option(
+#         "output.maf", "--output", "-o", help="Maf output file name."
+#     ),
+#     separator: str = typer.Option(
+#         "tsv",
+#         "--separator",
+#         "-sep",
+#         help="Specify a separator for delimited data.",
+#         callback=check_separator,
+#     ),
+# ):
+#     # prep maf
+#     mafa = MAFFile(maf, separator)
+#     typer.secho(
+#         f"Tagging MAF with column for presence of MET variant", fg=typer.colors.BRIGHT_GREEN
+#     )
+#     mafa = mafa.tag_met_variant()
+#     typer.secho(f"Writing Delimited file: {output_maf}", fg=typer.colors.BRIGHT_GREEN)
+#     mafa.to_csv(f"{output_maf}".format(outputFile=output_maf), index=False, sep="\t")
+#     return 0
+
+
+# @app.command(
+#     "TERT_variant", help="Tag a variant in a MAF file if it is classified as a TERT variant."
+# )
+# def TERT_variant(
+#     maf: Path = typer.Option(
+#         ...,
+#         "--maf",
+#         "-m",
+#         exists=True,
+#         file_okay=True,
+#         dir_okay=False,
+#         writable=False,
+#         readable=True,
+#         resolve_path=True,
+#         help="MAF file to tag",
+#     ),
+#     output_maf: Path = typer.Option(
+#         "output.maf", "--output", "-o", help="Maf output file name."
+#     ),
+#     separator: str = typer.Option(
+#         "tsv",
+#         "--separator",
+#         "-sep",
+#         help="Specify a separator for delimited data.",
+#         callback=check_separator,
+#     ),
+# ):
+#     # prep maf
+#     mafa = MAFFile(maf, separator)
+#     typer.secho(
+#         f"Tagging Maf with column for presence of TERT variant", fg=typer.colors.BRIGHT_GREEN
+#     )
+#     mafa = mafa.tag_tert_variant()
+#     typer.secho(f"Writing Delimited file: {output_maf}", fg=typer.colors.BRIGHT_GREEN)
+#     mafa.to_csv(f"{output_maf}".format(outputFile=output_maf), index=False, sep="\t")
+#     return 0
+
+
+@app.command(
+    "tag_by_maf",
+    help="Tag a variant in a MAF file based on criterion provided by input file",
+)
+def tag_by_maf(
```

Comment:
change name to `tag by_rules`

Reply:
7ab98306a2b85867c65c4923919d82f784a9165f